feat(curriculum): add case converter project to teach list comprehension (#52569)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/20-upcoming-python/learn-list-comprehension-by-building-a-case-converter-program/657efa642593c5746acc5c81.md

---
id: 657efa642593c5746acc5c81
title: Step 5
challengeType: 20
dashedName: step-5
---

# --description--

> Inside the if statement body, you need to convert any uppercase character to lowercase and prepend an underscore to that.

Reviewer: ```suggestion
Inside the if statement body, you need to convert any uppercase character to lowercase and prepend an underscore to this lowercase character.
```

> Use the `lower()` string method to convert uppercase characters to lowercase characters. You can then concatenate an underscore to that using the plus sign.

Reviewer: ```suggestion
Use the `.lower()` string method to convert uppercase characters to lowercase characters. You can then concatenate an underscore to the character using the plus sign.
```

```python
'_' + char.lower()
```

> Keep the modified character inside a variable named `converted_character` inside the if statement body.

Reviewer: ```suggestion
Assign the modified character to a variable named `converted_character` inside the if statement body.
```
